web/public/locales/sv.json

```
@@ -1843,7 +1843,7 @@
       "zoneName": "Zimbabwe"
     },
     "US": {
-      "zoneName": "Amerikas förenta stater"
+      "zoneName": "Sammanhängande Amerikas förenta stater"
```

Not sure if people actually write it this way and it might be a little long so I suggest we just use "Sammanhängande USA" to make it a bit shorter.
```suggestion
      "zoneName": "Sammanhängande USA"
```